diff -Naur rocksdb-7.7.3/include/rocksdb/utilities/checkpoint.h rocksdb-7.7.3-patched/include/rocksdb/utilities/checkpoint.h
--- rocksdb-7.7.3/include/rocksdb/utilities/checkpoint.h	2022-10-11 12:18:26.000000000 -0700
+++ rocksdb-7.7.3-patched/include/rocksdb/utilities/checkpoint.h	2025-11-03 15:58:49.834967673 -0800
@@ -8,6 +8,7 @@
 #pragma once
 #ifndef ROCKSDB_LITE
 
+#include <cstdint>
 #include <string>
 #include <vector>
 #include "rocksdb/status.h"
